spk/homeassistant/Makefile

```diff
@@ -5,7 +5,7 @@ SPK_ICON = src/homeassistant.png
 
 HACS_VERS = 1.31.0
 
-SPK_DEPENDS = "python310"
+SPK_DEPENDS = "python310>=3.10.12-18"
```

Comment:
IMHO this is not required.
I successfully tested homeassistant v2023.1.7-21 with python 3.10.12-16.

Reply:
I thought this would make a demarcation point on all python packages whereas from this point on they've been tested to work with openssl3.